rancher2/data_source_rancher2_global_dns_provider_test.go

```diff
@@ -0,0 +1,51 @@
+package rancher2
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+const (
+	testAccRancher2GlobalDNSProviderDataSourceType = "rancher2_global_global_dns_provider"
```

Comment:
`testAccRancher2GlobalDNSProviderDataSourceType ` should be `rancher2_global_dns_provider`